docs/detections/rules-ui-create.asciidoc

```diff
@@ -517,12 +520,12 @@ minutes at 14:03 but it does not run until 14:04, it will run again at 14:09.
 [[rule-action-variables]]
 ==== Alert notification placeholders
 
-You can use http://mustache.github.io/[mustache syntax] to add the following placeholders to <<rule-notifications, rule action>> fields:
+You can use http://mustache.github.io/[mustache syntax] to add variables to notification messages. The action frequency you choose determines the variables you can select from.   
+
+The following variables can be passed if the rule’s action frequency is a summary of alerts or per alert: 
+
+NOTE: Refer to {kibana-ref}/rule-action-variables.html#alert-summary-action-variables[Action frequency: Summary of alerts] to learn about additional variables that can be passed if the rule's action frequency is summary of alerts. 
```

Comment:
```suggestion
NOTE: Refer to {kibana-ref}/rule-action-variables.html#alert-summary-action-variables[Action frequency: Summary of alerts] to learn about additional variables that can be passed if the rule's action frequency is `summary of alerts`. 
```
IMO some kind of visual reminder that "Summary of alerts" is a setting/option/term would be helpful, maybe quotes, maybe capitals, maybe `code`

Reply:
Good point! I'll bold this since it's referring to the UI option.

```suggestion
NOTE: Refer to {kibana-ref}/rule-action-variables.html#alert-summary-action-variables[Action frequency: Summary of alerts] to learn about additional variables that can be passed if the rule's action frequency is **Summary of alerts**. 
```